crates/egui/src/context.rs

```diff
@@ -33,15 +25,23 @@ use crate::{
     os::OperatingSystem,
     output::FullOutput,
     pass_state::PassState,
-    resize, response, scroll_area,
+    plugin, resize, response, scroll_area,
     util::IdTypeMap,
     viewport::ViewportClass,
 };
-
-use self::{hit_test::WidgetHits, interaction::InteractionSnapshot};
-#[cfg(feature = "accesskit")]
-use crate::IdMap;
-use crate::input_state::SurrenderFocusOn;
+use emath::{GuiRounding as _, OrderedFloat};
+use epaint::{
+    ClippedPrimitive, ClippedShape, Color32, ImageData, ImageDelta, Pos2, Rect, StrokeKind,
+    TessellationOptions, TextureAtlas, TextureId, Vec2,
+    emath::{self, TSTransform},
+    mutex::RwLock,
+    stats::PaintStats,
+    tessellator,
+    text::{FontInsert, FontPriority, Fonts},
+    vec2,
+};
+use std::any::TypeId;
+use std::{borrow::Cow, cell::RefCell, panic::Location, sync::Arc, time::Duration};
```

Comment:
nit: In most places we try to follow this include order:

* std::
* 3rdparty::
* 1stparty::